incubator/hnc/pkg/reconcilers/object.go

```diff
@@ -180,9 +193,29 @@ func (r *ObjectReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 			return resp, err
 		}
 	}
+
+	// We need to compute NumPropagatedObjects and show it in the status of the `config`
+	// singleton even in ignore mode.
```

Comment:
I might misunderstand that. We agreed on only reporting `NumSourceObjects` for `propagate` mode.  I thought that meant we would report `NumPropagatedObjects` for all modes. I will delete the logic if we decide to follow the same rule and only report  `NumPropagatedObjects` for `propagate` mode as well.

WDYT?

Reply:
Actually, I think it makes sense to not record `NumPropagatedObjects` in `ignore` mode since we also use that to "terminate" an object reconciler.

I guess the only question is that if we want to record `NumPropagatedObjects` in `remove` mode. The number should go to 0 eventually. I am not sure if users are interested in the process.

Reply:
As discussed offline. I supported `NumPropagatedObjects` in `propagate` and `remove` modes.